google/cloud/firestore_v1/order.py

```diff
@@ -32,7 +32,8 @@ class TypeOrder(Enum):
 
     @staticmethod
     def from_value(value):
-        v = value.WhichOneof("value_type")
+        # return TypeOrder.OBJECT
+        v = value._pb.WhichOneof("value_type")
```

Comment:
I'm intrigued by this change. I was of the impression that the general change was in the direction `<type>._pb` to `<type>`, but this moving in the opposite direction. Is there a reason behind this?

Reply:
After reviewing the rest of the PR I think I have a better understanding of the restructure and I'm no longer confused :)